service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/flexibleresource/CloneControlledFlexibleResourceFlight.java

```diff
@@ -42,7 +42,6 @@ public CloneControlledFlexibleResourceFlight(FlightMap inputParameters, Object b
         inputParameters,
         WorkspaceFlightMapKeys.ResourceKeys.RESOURCE,
         JobMapKeys.AUTH_USER_INFO.getKeyName(),
-        WorkspaceFlightMapKeys.ResourceKeys.CLONING_INSTRUCTIONS,
```

Comment:
Removed this unnecessary restriction: The resolved cloning instructions are calculated later regardless.

If the cloning instructions are not provided in the input parameters, then we use the source flex resource cloning instructions.